aptos-move/framework/aptos-framework/sources/coin.move

```diff
@@ -124,11 +124,23 @@ module aptos_framework::coin {
         amount: u64,
     }
 
+    #[event]
+    struct Deposit has drop, store {
```

Comment:
Would `Deposit` and `Withdraw` need the `CoinType` parameter? Otherwise, one can't tell the coin type from the `Deposit` (`Withdraw`) events? (cc: @movekevin )

Reply:
I think you are right.